biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -98,6 +98,12 @@
 	String							DEPLOYREPO									= "-deployrepo";
 	String							DIGESTS										= "-digests";
 	String							DSANNOTATIONS								= "-dsannotations";
+	String							DSANNOTATIONS_OPTIONS						= "-dsannotations-options";
+	// -dsannotations-options values
+	String							DSANNOTATIONS_OPTIONS_INHERIT				= "inherit";
```

Comment:
I realized that when you renamed these option value constants (as I asked) that no other code has to change. This is because the "real" constant is the name of the DSAnnotations.Flag enum values. So given this, we should either not have these string constants _or_ the string constants values must come from the enum:

    String DSANNOTATIONS_OPTION_INHERIT = DSAnnotations.Flag.inherit.name();

(Assuming that will compile.)

I am leaning to just removing the constants.